xwiki-platform-core/xwiki-platform-security/xwiki-platform-security-requiredrights/xwiki-platform-security-requiredrights-ui/src/main/resources/js/security/requiredrights/requiredRightsDialog.js

```diff
@@ -444,6 +446,128 @@ define('xwiki-requiredrights-dialog', [
 
             this.analysisResultsContainer.appendChild(panelGroup);
         }
+
+        /**
+         * @returns {number} the minimum width of the rights list in horizontal layout
+         */
+        #computeMinWidth()
```

Comment:
The name of the method also does not state clearly that it has side effects on `this.rightsList`